crates/ruff_linter/src/rules/airflow/rules/moved_to_provider_in_3.rs

```diff
@@ -0,0 +1,150 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, ViolationMetadata};
+use ruff_python_ast::{Expr, ExprAttribute};
+use ruff_python_semantic::Modules;
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+
+#[derive(Debug, Eq, PartialEq)]
+enum Replacement {
+    ProviderName(String, String, String),
+    ImportPathMoved(String, String, String, String),
+}
```

Comment:
I'd prefer if we use variants with named fields. It's kind of hard to guess what the 4 Strings in `ImportPathMoved` mean
```suggestion
enum Replacement {
    ProviderName { name: String, provider_name: String, version: String},
    ImportPathMoved { original_path: String, new_path: String, provider_name: String, provider_version: String },
}
```

Reply:
Sure! can't believe I missed it 🤦‍♂️ Thanks for your suggestion! Will update it

Reply:
fixed. Thanks!